packages/suite/src/views/wallet/transactions/components/CoinjoinAccountWithoutTor.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { useActions } from '@suite-hooks';
+import { Button, Card, H2, Image, P, variables } from '@trezor/components';
+import { Translation } from '@suite-components/Translation';
+import * as routerActions from '@suite-actions/routerActions';
+import { SettingsAnchor } from '@suite-constants/anchors';
+
+const Container = styled(Card)`
+    align-items: center;
+    text-align: center;
+    padding: 40px 120px;
+`;
+
+const Heading = styled(H2)`
+    margin: 36px auto 18px;
+`;
+
+const AccountDescription = styled(P)`
+    font-size: ${variables.FONT_SIZE.SMALL};
+`;
+
+export const CoinjoinAccountWithoutTor = () => {
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+
+    const action = (...gotoArgs: Parameters<typeof goto>) => {
+        goto(...gotoArgs);
+    };
+
+    return (
+        <Container>
+            <Image image="TOR_DISABLE_QUESTION" width={300} />
+
+            <Heading>
+                <Translation id="TR_COINJOIN_ACCOUNT_TOR_DISABLE_HEADING" />
+            </Heading>
+
+            <AccountDescription>
+                <Translation id="TR_TOR_REQUEST_ENABLE_FOR_COIN_JOIN_TITLE" />
+            </AccountDescription>
+
+            <Button onClick={() => action('settings-index', { anchor: SettingsAnchor.Tor })}>
+                <Translation id="TR_TOR_ENABLE" />
+            </Button>
+        </Container>
```

Comment:
Here when the user comes to CoinJoin account and goes to enable Tor, I am not sure if probably the clients and backend instances should be recreated since they were disposed before. 

WDYT @szymonlesisz ?